PyDSS/defaults/simulation.toml

```diff
@@ -14,12 +14,9 @@
 # Active Scenario- [String] - Project scenario to use
 # DSS File- [String] - The main OpenDSS file
 [Project]
-"Start Year" = 2017
-"Start Day" = 1
-"Start Time (min)" = 0.0
-"End Day" = 1
-"End Time (min)" = 1439.0
-"Date offset" = 0
+"Start time" = "1/1/2020 00:00:00"
+"Simulation duration (min)" = 1440.0
+"Loadshape start time" = "1/1/2020 00:00:00"
```

Comment:
These timestamps need to use the new format specified by DATE_FORMAT.

Reply:
fixed